react-app/src/pages/LoginPage.js

```diff
@@ -1,14 +1,15 @@
 import React, { useEffect, useState } from 'react'
-import { useHistory } from 'react-router-dom'
 import { createUser, ADMIN_USER } from '../domain/User'
 import RoundedAvatar from '../components/RoundedAvatar'
 import LoadingAnimation from '../components/LoadingAnimation'
-import firebase from 'firebase'
+import firebase from '../firebase'
```

Comment:
Right now one of the third-party dependencies is leaking into your app's codebase. In such a case, it would be difficult to remove the dependency or replace it (more often scenario). Let's discuss it in the meeting

Reply:
- [ ] Discuss abstraction over dependencies